webpack.config.js

```diff
@@ -50,6 +50,14 @@ module.exports = function(env = 'production') {
           exclude: /node_modules/,
         },
 
+        {
+          test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
+          include: [
+            path.resolve(__dirname, "packages/Icon")
```

Comment:
Going to whitelist which folders will use `svgr`? We don't want all svgs to use svgr?

Reply:
Not necessarily. Initially I was gonna specify the configuration here which we won't want to apply to all SVGs (given the icon-specific template). Since we aren't, I could justify removing this for the time being.

Reply:
Hmm....

Yeah I'm leaning towards removing it such that the assumption is this behavior is true for all usages. i.e. You assume this rule is applied universally from the start, not that it starts as an exceptional opt-in behavior. 

If we need to customize our svg consumption then we can edit the webpack config at that point.

Reply:
Fair enough!